src/pac_scan.ll

```diff
@@ -44,10 +44,26 @@ int char_token(int tok)
 
 void include_file(const char *filename);
 
-string dirname(std::string_view path)
-{
-  return std::filesystem::path(path).parent_path().string();
-}
+/**
+ * Portable implementation of dirname(3). Adapted from the version in musl. See
+ * http://git.musl-libc.org/cgit/musl/tree/COPYRIGHT for licensing information
+ * for the original version.
+ *
+ * TODO: This probably isn't as efficient as it could be, but that's fine because
+ * this code is only run during the Zeek build and it doesn't matter that much .
```

Comment:
Maybe point to https://github.com/BlankOn/musl/blob/master/src/misc/dirname.c?  Also, veeery important:
```suggestion
 * this code is only run during the Zeek build and it doesn't matter that much.
```